react-native/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "RNSDKTester",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "lint": "eslint .",
+    "start": "react-native start",
+    "update-deps": "node node_modules/@jitsi/react-native-sdk/update_dependencies.js"
+  },
+  "dependencies": {
+    "@amplitude/react-native": "2.7.0",
+    "@giphy/react-native-sdk": "2.3.0",
+    "@jitsi/react-native-sdk": "file:../jitsi-meet/react-native-sdk/jitsi-react-native-sdk-0.1.0.tgz",
+    "@react-native-async-storage/async-storage": "1.18.2",
+    "@react-native-community/clipboard": "1.5.1",
+    "@react-native-community/netinfo": "7.1.7",
+    "@react-native-community/slider": "4.1.12",
+    "@react-native-google-signin/google-signin": "7.0.4",
+    "react": "*",
```

Comment:
These cannot be set to * on a project, since updating RN is not just updating the number. Please keep them to the initial values when you created the app.